app/src/org/commcare/adapters/MenuAdapter.java

```diff
@@ -178,12 +180,12 @@ private void setupAudioButton(int rowId, AudioPlaybackButton audioPlaybackButton
         } else {
             if (audioPlaybackButton != null) {
                 audioPlaybackButton.modifyButtonForNewView(viewId, audioURI, false);
-                ((LinearLayout)audioPlaybackButton.getParent()).removeView(audioPlaybackButton);
+                audioPlaybackButton.setVisibility(View.GONE);
```

Comment:
@amstone326 Where in HQ do we add audio for a menu ?  I want to test once for that this change won't cause any regressions!

Reply:
If you select the options "Gear" icon on a menu the audio upload widget should be right there:

![screen shot 2017-09-06 at 3 13 35 pm](https://user-images.githubusercontent.com/2293064/30130059-fc38db8a-9315-11e7-8eb8-18df753b290c.png)

Reply:
@shubham1g5 Let me know when you're done testing this; I'm good to merge once you have